cluster-provision/images/vm-image-builder/build-containerdisk.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$BASH_SOURCE[0]")"
+  pwd
+)"
+
+function cleanup() {
+  echo "[build-container-disk] cleanup"
+  rm -f Dockerfile
+
+  if [ $? -ne 0 ]; then
+    rm -f "$tar_file"
+    docker image rm "$full_image_tag"
+  fi
+}
+
+image_name=$1
+tag=$2
+vm_image_file=$3
+
+export TEMPLATE_DOCKERFILE=${TEMPLATE_DOCKERFILE:-$SCRIPT_DIR/Dockerfile.containerdisk}
+
+parent_dir=$(basename "$(dirname "$vm_image_file")")
+tar_file="${parent_dir}/${image_name}-${tag}.tar"
+full_image_tag="${image_name}:${tag}"
+
+trap 'cleanup' EXIT
+
+export IMAGE=${vm_image_file}
```

Comment:
Why do you need to export this?

Reply:
You right it should not be exported, I will remove export in following commits

Reply:
Sorry I mislead you, `IMAGE` should be exported because we use `envsubst` to replace the placeholder with the image file path